tests/test_main.py

```diff
@@ -22,87 +32,115 @@ def created_person_list(people_data):
 
 
 def test_person_class_attribute_people_exists():
-    assert hasattr(Person, 'people'), (
-        "Class Person should have class attribute 'people'"
-    )
-    assert len(Person.people) == 0, (
-        "Initial length of 'Person.people' should equal to 0"
-    )
+    assert hasattr(
+        Person, "people"
+    ), "Class Person should have class attribute 'people'"
+    assert (
+        len(Person.people) == 0
+    ), "Initial length of 'Person.people' should equal to 0"
 
 
-@pytest.mark.parametrize(
-    'name,age',
-    [
-        ('Ross', 30),
-        ('Joey', 29)
-    ]
-)
+@pytest.mark.parametrize("name,age", [("Ross", 30), ("Joey", 29)])
 def test_person_class_name_age(name, age):
     person_inst = Person(name, age)
-    assert person_inst.name == name, (
-        "Person instance should have attribute 'name'"
-    )
-    assert person_inst.age == age, (
-        "Person instance should have attribute 'age'"
-    )
+    assert person_inst.name == name, "Person instance should have attribute 'name'"
+    assert person_inst.age == age, "Person instance should have attribute 'age'"
 
 
 def test_create_person_list_all_persons(people_data, created_person_list):
-
     assert all(isinstance(person, Person) for person in created_person_list), (
         "All elements in result of 'create_person_list' should be instance "
         "of Person class"
     )
-    assert len(created_person_list) == len(people_data), (
-        "Length of initial list should equal to length of function result"
-    )
+    assert len(created_person_list) == len(
+        people_data
+    ), "Length of initial list should equal to length of function result"
 
 
 def test_create_person_list_order(people_data, created_person_list):
-   assert [person_dict['name'] for person_dict in people_data] == [
-       person.name for person in created_person_list
-   ], (
-        "Order in function result should be the same"
-    )
+    assert [person_dict["name"] for person_dict in people_data] == [
+        person.name for person in created_person_list
+    ], "Order in function result should be the same"
 
 
 def test_create_person_list_has_wife(people_data, created_person_list):
-    assert hasattr(created_person_list[0], 'wife'), (
+    assert hasattr(created_person_list[0], "wife"), (
         f"Person with 'name' {created_person_list[0].name} should have "
         f"attribute 'wife' with name {people_data[0].wife.name}"
```

Comment:
There is a mistake in the assertion message. `people_data[0].wife.name` should be `people_data[0]['wife']` since `people_data` is a list of dictionaries and 'wife' is a key in the dictionary.